bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,325 @@
+= WFLY-13305: WildFly bootable jar
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+We are proposing a way to package server and application inside a bootable jar. The jar contains everything needed to run the server.
+The packaged server can benefit from Galleon tooling and be trimmed down to user needs (thanks to Galleon layers).
+Such executable jar can then be run on bare metal and in cloud execution contexts.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13305[WFLY-13305]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1385[EAP7-1385]
+
+* https://issues.redhat.com/browse/WFCORE-4895[WFCORE-4895]
+
+* https://issues.redhat.com/browse/WFGP-169[WFGP-169]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[ ] Engineering
+
+[X] QE
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/wildfly-maven-plugin[WildFly maven plugins]
+
+* https://github.com/wildfly/quickstart[WildFly Quickstarts]
```

Comment:
Actually I don't know how quickstarts should evolve and if it is a good idea to evolve them. I added this project because it seemed to make sense, but now I am unsure. They seem bound to the WildFly EE way of deploying applications. This could create confusion if we mix the bootable jar with legacy WildFly. Perhaps we need new quickstarts for Eclipse Microprofile that would use the bootable jar?

Reply:
Yep, that's my Point @jfdenise. I see there's no decision on this yet so I think that the whole "Quickstarts for bootable JAR" topic should be tracked separately, as it was for MicroProfile specs integration. WDYT?

Reply:
BTW - In such case if we agree on removing the item from here, I'd suggest to add a note for it as a NON-requirement.

Reply:
Let's wait a bit, we are going to discuss this topic, I will update this point accordingly.

Reply:
Yep, makes sense to me, thanks!

Reply:
Thank-you, Quickstarts will be handled in a separated WildFly RFE, I added a note in "Related Issues"